src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/wwwroot/styles/colors.scss

```diff
@@ -7,7 +7,7 @@ $VeryLightPink: #f5f3f3;
 $LightGray: #fbfbfb;
 $LightGreen: #4d8636;
 $DarkGreen: #3b653d;
-$LightBlue: #2a3bfa;
+$LightBlue: #537EC5;
```

Comment:
are you sure about this chagne?

Reply:
I know this change will affect other pages as well, but I think the old color was a little dark